originals/m.yaml

```diff
@@ -364,15 +364,40 @@
     - Arcade
 
 - name: Minecraft
+  names:
+  - 'Minecraft: Java Edition'
+  - 'Minecraft: Windows 10 Edition'
+  - 'Minecraft: Xbox 360 Edition'
+  - 'Minecraft: New Nintendo 3DS Edition'
+  - 'Minecraft: Wii U Edition'
+  - 'Minecraft: Playstation 3 Edition'
+  - 'Minecraft: Playstation Vita Edition'
```

Comment:
I don't think it's necessary to add these alternate names; the platform information is enough

Reply:
I’m thinking about renaming it to
```
- 'Minecraft: Java Edition'
- 'Minecraft: Legacy Console Edition'
- 'Minecraft: Bedrock Edition'
- 'Minecraft: Pi Edition'
```

Which would make sense for the purpose of the site, since features and gameplay is not neccesarily the same. And they share some platforms.

So different projects could try to clone different editions.

Java Edition: (formerly just Minecraft)
- Windows
- MacOS
- Linux

Legacy Console Edition: (initial console ports by a third party)
- Xbox 360
- Xbox One
- PS3
- PS4
- PS Vita
- Wii U
- Nintendo Switch

Bedrock Edition: (C++ recreation of the engine)
- Android/Fire OS
- iOS
- Windows 10
- Xbox One
- Nintendo Switch
- PlayStation 4

Pi Edition is a free version for ARM only. With cut down functionality, but programming language APIs.

Reply:
The names here are to help people find games that can appear under very different names, for example "Resident Evil 2"/"Biohazard". For minecraft this isn't necessary, because they all have the "minecraft" name in common.
If there are clones that only implement say the legacy console features, this can go in the `info` field.
Having too many names makes it appear bloated in the webpage.